presto-main/src/main/java/com/facebook/presto/util/DateTimeUtils.java

```diff
@@ -548,4 +552,52 @@ else if (parsePos < 0) {
             return ~bestInvalidPos;
         }
     }
+
+    public static SqlTimestamp sqlTimestampOf(int year,
```

Comment:
It can be in presto-main/testing as those are helpers for different modules tests as well.